package.json

```diff
@@ -58,15 +35,11 @@
     "watch": "encore dev --watch",
     "build": "encore dev",
     "build:prod": "encore production",
-    "gulp": "gulp build",
     "lint": "yarn lint:js",
-    "lint:js": "eslint gulpfile.babel.js src/Sylius/Bundle/AdminBundle/gulpfile.babel.js src/Sylius/Bundle/ShopBundle/gulpfile.babel.js src/Sylius/Bundle/UiBundle/Resources/private/js src/Sylius/Bundle/AdminBundle/Resources/private/js src/Sylius/Bundle/ShopBundle/Resources/private/js",
     "postinstall": "semantic-ui-css-patch"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Sylius/Sylius.git"
-  },
```

Comment:
Did you intend to remove `author` and `license` fields?
I'm not familiar with npm packages publishing rules, so don't know a possible reason.

Reply:
These two entries were duplicated 😅